alignak/daemons/arbiterdaemon.py

```diff
@@ -197,17 +198,25 @@ def get_initial_broks_from_satellitelinks(self):
                 brok = sat.get_initial_status_brok()
                 self.add(brok)
 
-    def load_external_command(self, ecm):
-        """Set external_command attribute to the external command manager
+    def set_external_commands_manager(self, ecm, commands_file=None):
+        """Set our external_commands_manager property to the external command manager
         and fifo attribute to a new fifo fd
 
+        If the fifo parameter is nont None, it must contain the commands file name and this
```

Comment:
I will replace this with a log in the next version because I disabled this function in the ExternalCommandManager class ...